Sources/Sentry/SentryHub.m

```diff
@@ -100,4 +105,20 @@ - (void)configureScope:(void(^)(SentryScope *scope))callback {
     }
 }
 
+- (BOOL)doInstallIntegrations {
+    // TODO(fetzig) change this. instead of skipping whenever integrations have already been installed, check the integrations one by one, and skip those that are already installed.
+    if (SentrySDK.currentHub.installedIntegrations.count > 0) {
+        [SentryLog logWithMessage:@"[SentryHub doInstallIntegrations] there are already installed integrations. skipping isntall." andLevel:kSentryLogLevelError];
+        return NO;
+    }
```

Comment:
Can we check here for every specific integration? This is important.

Reply:
perfect. wasn't sure. done.